tests/test_rawpy.py

```diff
@@ -0,0 +1,20 @@
+"""Tests for rawpy plugin
+"""
+
+import os
+import rawpy
+import imageio.v3 as iio
+
+
+def test_nef_local(test_images):
+    """Test for reading .nef file from .test_images dir.
+    """
+    # Check image path
+    im_path = str(test_images / "Nikon.nef")
+    assert os.path.exists(im_path)
```

Comment:
```suggestion
```

No need to assert this. We are not testing the correctness of our testing fixtures here. (We assume they work as they should)

All assertions should be done to assert correctness of the RawPy plugin. This way pytest will report "failed" if the test fails and "Error" if something in the supporting logic fails.

Reply:
Ok, thanks! I have updated it now.